taildrop/resume.go

```diff
@@ -64,7 +64,7 @@ func (m *Manager) PartialFiles(id ClientID) (ret []string, err error) {
 	if m == nil || m.opts.Dir == "" {
 		return nil, ErrNoTaildrop
 	}
-	if m.opts.DirectFileMode && m.opts.AvoidFinalRename {
+	if m.opts.DirectFileMode {
```

Comment:
This used to depend on `AvoidFinalRename` being true, which was only true for MacOS, right? Should we just remove this block completely?